### PR TITLE
Readme update for Segment kotlin Destination

### PR DESCRIPTION
**Pull request by rishabh-engg-sdk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,21 +123,21 @@ For showing Push notifications there are 2 important steps:
...
-MoEngage moEngage = new MoEngage.Builder(this, "XXXXXXXXXX")
-            .configureNotificationMetaData(new NotificationConfig(R.drawable.small_icon, R.drawable.large_icon))
-            .configureFcm(FcmConfig(false))
-            .enablePartnerIntegration(IntegrationPartner.SEGMENT)
-            .build();
-MoEngage.initialiseDefaultInstance(moEngage);
+```kotlin
+val moEngage = MoEngage.Builder(this, "XXXXXXXX")
+    .configureNotificationMetaData(NotificationConfig(R.drawable.small_icon, R.drawable.large_icon, R.color.notiColor, null, true, isBuildingBackStackEnabled = false, isLargeIconDisplayEnabled = true))
+    .configureFcm(FcmConfig(false))
```
Comment (umangmoe) [inline README.md:129]: Partner integration not enabled.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,21 @@
 ![Logo](/.github/logo.png)
 
-analytics-android-integration-moengage
+analytics-kotlin-destination-moengage
```
Comment (umangmoe) [inline README.md:3]: We need a note for the `moengage-segment-integration` around deprecation and link to the integration document

File: README.md (hunk 1 above)
Comment (rishabh-engg-sdk) [reply README.md:133]: Done

File: README.md (hunk 2 above)
Comment (rishabh-engg-sdk) [reply README.md:3]: Done

**Pull request merged**